package.json

```diff
@@ -35,50 +23,73 @@
     "url": "https://github.com/effect-ts/docgen/issues"
   },
   "homepage": "https://github.com/effect-ts/docgen",
+  "files": [
+    "dist",
+    "src",
+    "schema.json"
+  ],
+  "scripts": {
+    "prepare": "pnpm clean && pnpm build",
+    "update-version": "changeset version && pnpm install --no-frozen-lockfile",
+    "lint": "eslint src test",
+    "lint-fix": "eslint src test --fix",
+    "coverage": "vitest run --coverage",
+    "check": "tsc -b tsconfig.json",
+    "test": "vitest",
+    "clean": "rimraf dist coverage .tsbuildinfo",
+    "build": "tsc -b tsconfig.build.json",
+    "schema": "tsx scripts/gen-schema.ts",
+    "docgen": "tsx src/bin.ts"
+  },
   "dependencies": {
-    "@effect/platform-node": "^0.27.0",
-    "@effect/schema": "^0.46.1",
+    "@babel/core": "^7.23.2",
```

Comment:
is this an actual dep?

Reply:
Oh no. I just added that when I played with the transform for the dynamic imports. Good catch.